lib/walkman_server.ex

```diff
@@ -94,4 +94,22 @@ defmodule WalkmanServer do
     Path.relative_to("test/fixtures/walkman", File.cwd!()) |> File.mkdir_p()
     filename(test_id) |> File.write!(:erlang.term_to_binary(Enum.reverse(tests)), [:write])
   end
+
+  defp args_match?({m, f, a}, {m2, f2, a2}) do
```

Comment:
Actually you can also get rid of this one haha

Reply:
Like this? You would still need the compare, or we can move that up to the place of the `args_match?` usage. 
```
  defp args_match?(args, args2) do
    normalize_args(args) == normalize_args(args2)
  end
```

Reply:
I didn't mean like this (forgot about the compare) but this is good.